src/extension.ts

```diff
@@ -37,7 +47,15 @@ export function activate(context: vscode.ExtensionContext) {
 
             const text = editor.document.getText()
             // JSON.parse(text)
-            const path = jsPathTo(text, editor.document.offsetAt(editor.selection.active), nonQuotedKeyRegex)
+            let path = jsPathTo(text, editor.document.offsetAt(editor.selection.active), nonQuotedKeyRegex)
+
+            if (fileNameAsPrefix && editor.document.fileName) {
+                const fileName = parse(editor.document.fileName).base;
+                const fileNameWithoutExtension = fileName.substring(0, fileName.lastIndexOf('.'))
+                
+                path = `${fileNameWithoutExtension}${prefixSeparator}${path}`
```

Comment:
When calling the function in a not yet saved file - you still get a prefix separator, e.g. `:attributes[0].value`. Is it desirable?

Reply:
Good point, I added a check for this case.